packages/rspack/src/MultiStats.ts

```diff
@@ -65,32 +62,36 @@ export default class MultiStats {
 			warningsCount: children.every(o => o.warningsCount),
 			errors: children.every(o => o.errors),
 			warnings: children.every(o => o.warnings),
-			children
+			children,
+			context: "",
+			version: ""
```

Comment:
Is this necessary?

Reply:
In order to prevent errors in the TS type.

Reply:
These attributes were used, but we did not give them initial values.

Reply:
You can notice `childOptions` attributes `context` and `version` below: 
https://github.com/web-infra-dev/rspack/blob/d686b72a22e1f44bc7e00d4837d50251176ea177/packages/rspack/src/MultiStats.ts#L71-L90

Reply:
not a good code now due to webpack laziness, 

I think we need to replace `options: any` in the future